pkg/agent/openflow/client.go

```diff
@@ -294,6 +294,11 @@ type Client interface {
 	UninstallTrafficControlReturnPortFlow(returnOFPort uint32) error
 
 	InstallMulticastGroup(ofGroupID binding.GroupIDType, localReceivers []uint32) error
+	// InstallHostUplinkFlows installs flows to forward packet between uplinkPort and hostPort.
+	InstallHostUplinkFlows(hostInterfaceName string, hostPort int32, uplinkPort int32) error
+	// UninstallHostUplinkFlows removes the flows installed to forward packet between uplinkPort and hostPort.
+	UninstallHostUplinkFlows(hostInterfaceName string) error
+	InstallHostReservedFlows(proto binding.Protocol, ipnet *net.IPNet, ip net.IP, port uint16, isIngress bool) error
```

Comment:
Add comments for this one too.

Reply:
Should we name it InstallPolicyBypassFlows to be clear? I know you have another func called bypassPolicyRuleCheckFlows, which I am thinking renaming to skipPolicyRuleCheck.

My two names are not very good either. So I am fine if you prefer to keep the current ones.